build.sbt

```diff
@@ -22,14 +22,16 @@ version := sys.props
   }
   .getOrElse("LOCAL-SNAPSHOT")
 
-val slf4j = "org.slf4j" % "slf4j-api" % "1.7.21"
+val AkkaVersion = "2.6.4"
+
+val slf4j = "org.slf4j" % "slf4j-api" % "1.7.26"
 val logback = "ch.qos.logback" % "logback-classic" % "1.2.3"
-val amazonKinesisClient = "com.amazonaws" % "amazon-kinesis-client" % "1.9.3"
-val scalaKinesisProducer = "com.streetcontxt" %% "kpl-scala" % "1.0.5"
-val scalaTest = "org.scalatest" %% "scalatest" % "3.0.1"
-val scalaMock = "org.scalamock" %% "scalamock-scalatest-support" % "3.6.0"
-val akkaStream = "com.typesafe.akka" %% "akka-stream" % "2.5.11"
-val akkaStreamTestkit = "com.typesafe.akka" %% "akka-stream-testkit" % "2.5.11"
+val amazonKinesisClient = "com.amazonaws" % "amazon-kinesis-client" % "1.8.8"
```

Comment:
Why did this version end up going down?

Reply:
Good catch! It seems to be a copy-paste error. Reverted back to the original version.